README.md

```diff
@@ -4,7 +4,7 @@
 [![Downloads](https://img.shields.io/npm/dm/napajs.svg)](https://www.npmjs.com/package/napajs)
 
 # Napa.js
-Napa.js is a multi-threaded JavaScript runtime built on [V8](https://github.com/v8/v8), which was originally designed to develop highly iterative services with non-compromised performance in Bing. As it evolves, we find it useful to complement [Node.js](https://nodejs.org) in CPU-bound tasks, with the capability of executing JavaScript in multiple V8 isolates and communicating between them. Napa.js is exposed as a Node.js module, while it can also be embedded in a host process without Node.js dependency.
+Napa.js is a multi-threaded JavaScript runtime built on [V8](https://github.com/v8/v8). Napa was originally designed to develop highly iterative services with non-compromised performance in Bing. As it evolved, we found it useful to complement [Node.js](https://nodejs.org) in CPU-bound tasks by executing JavaScript in multiple isolated V8 zones and administering communications among them. Even though Napa.js is a Node.js module, it can also be embedded in a host process without a Node.js dependency.
```

Comment:
"... by enabling JavaScript execution in multiple V8 isolates and communication between them." the problem here would be sentence parallelism (sorry to be a grammar geek :)). 

When you say "enabl**ing** \<something\> and communication \<something\>" there is no parallelism; by saying something like "enabl**ing** \<something\> and administer**ing** \<something\>" there is a _gerund_ parallelism.
Also _between_ is used when there are only two things; _among_ is used when there are two or more things.

So combining your suggestion and mine something like `... by enabling JavaScript execution in multiple V8 isolates and administering communication among them.`  sounds appropriate?